packages/mobile/src/exchange/actions.ts

```diff
@@ -10,6 +10,7 @@ export enum Actions {
   EXCHANGE_TOKENS = 'EXCHANGE/EXCHANGE_TOKENS',
   FETCH_TOBIN_TAX = 'EXCHANGE/FETCH_TOBIN_TAX',
   SET_TOBIN_TAX = 'EXCHANGE/SET_TOBIN_TAX',
+  WITHDRAW_CELO = 'EXCHANGE/WITHDRAW_CELO',
```

Comment:
Should this `WITHDRAW_CELO` action and it's saga go here or on the `transactions` folder?

Reply:
I think it would feel more natural in `transactions`, cause it's essentially a transfer and not an exchange, but having said that I do not have a strong opinion on this.